Gadget.Inspector/Services/Inspector.cs

```diff
@@ -15,14 +16,19 @@ namespace Gadget.Inspector.Services
     public class Inspector : BackgroundService
     {
         private readonly Channel<ServiceStatusChanged> _channel;
+        private readonly Channel<MachineHealthDataModel> _healthChannelWriter;
```

Comment:
_healthChannelWriter jest troche mylace bo to nie jest sam writer a caly channel